tds.go

```diff
@@ -1250,7 +1250,12 @@ initiate_connection:
 
 	if sess.routedServer != "" {
 		toconn.Close()
-		p.Host = sess.routedServer
+		// Need to handle case when routedServer is in "host\instance" format
+		routedParts := strings.SplitN(sess.routedServer, "\\", 2)
```

Comment:
Yeah, change the var everywhere so it builds.

Reply:
Didn't get the _var_ thing.

Should I put

		var routedParts []string = strings.SplitN(sess.routedServer, "\\", 2)